docs/advanced.md

```diff
@@ -337,3 +337,52 @@ Credential felix@krausefx.com:********* added to keychain.
 fastlane-credentials remove --username felix@krausefx.com
 password has been deleted.
 ```
+
+# Control configuration by lane and by platform
+
+In general, configuration files take only the first value given for a particular configuration item. That means that for an `Appfile` like the following:
+
+```ruby
+app_identifier "com.used.id"
```

Comment:
We do have tests for this. 👍 

https://github.com/fastlane/fastlane/blob/master/fastlane_core/spec/fixtures/ConfigFileRepeatedValueSet

This is what I was able to restore after reworking that original PR.